src/lib/core/components/Form/types/search.ts

```diff
@@ -0,0 +1,15 @@
+import {Spec} from '../../../types';
+
+import {FieldValue} from './value';
+
+export type SearchStore = {
```

Comment:
actually, u don't need this type, it's imported only once, u can replace it with Record